geojson/src/commonMain/kotlin/io/github/dellisd/spatialk/geojson/GeoJson.kt

```diff
@@ -1,12 +1,15 @@
 package io.github.dellisd.spatialk.geojson
 
+import kotlinx.serialization.json.JsonElement
+
 /**
  * A GeoJSON object represents a [Geometry], [Feature], or [collection of Features][FeatureCollection].
  *
  * @property bbox An optional bounding box used to represent the limits of the object's geometry.
  */
 interface GeoJson {
     val bbox: BoundingBox?
+    val foreignMembers: MutableMap<String, JsonElement>
```

Comment:
As this is always be defined by a backing field in the implementations, this property should be an immutable `Map`

Reply:
for public APIs I would recommend this 🙂